pyproject.toml

```diff
@@ -0,0 +1,32 @@
+[build-system]
+requires = ["hatchling"]
+build-backend = "hatchling.build"
+
+[project]
+name = "python_pkg_tmpl"
+version = "0.0.1"
+authors = [
+    { name="Minh Nguyen", email="minh.nguyen@uni-bremen.de" }
+]
+description = """
+A template Python package.
+"""
+readme = "README.md"
+requires-python = ">=3.10"
+classifiers = [
+    "Programming Language :: Python :: 3",
+    "Development Status :: 2 - Pre-Alpha",
+    "License :: OSI Approved :: MIT License"
+]
+dependencies = [
+  "platformdirs",
+]
+[project.optional-dependencies]
+test = [
+  "pytest",
+]
+[project.urls]
+"Homepage" = "https://github.com/secorolab/python-pkg-template"
```

Comment:
leave this as repository-name?

Reply:
does the linking work with just the repo name?

Reply:
I meant substituting python-pkg-template with repository-name

Reply:
I think this maybe handled by the build tool somehow, so I'd rather use a working link. We could also link to the AG.

Reply:
Ah, you mean so it works with GH for this repo?

Reply:
I think not on GitHub page, but on [PyPI project page](https://packaging.python.org/en/latest/guides/writing-pyproject-toml/#urls), in case it's released :smile: